src/schema/entities.yaml

```diff
@@ -93,6 +92,9 @@ echo:
     Multi-echo data MUST be split into one file per echo.
     Each file shares the same name with the exception of the `_echo-<index>`
     key/value.
+    Please note that the `<index>` denotes the number/index (in the form of a
+    nonnegative integer) of the echo not the echo time value which needs to be
+    stored in the field EchoTime of the separate JSON file.
```

Comment:
I am a bit lost as to how we typeset keywords... should they be monspaced or not?  In [01-magnetic-resonance-imaging-data.md](https://github.com/bids-standard/bids-specification/tree/master/src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md) you'll find
* `Echo Time` (monospaced, two words clearly referencing the DICOM Tag)
* Echo Times (not monospaced, two words)
* EchoTime (one word)
* `EchoTime` (one word, monospaced)

... anyway, if this "EchoTime" should be "`EchoTime`", then change it.

Reply:
I don't know the preference here either. It seems reasonable to monospace metadata fields though, so I'll do that.

Reply:
```suggestion
    stored in the field `EchoTime` of the separate JSON file.
```